server/models/Profile.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require("mongoose");
```

Comment:
Some of the code most likely belongs to a previous PR, please update the base of the PR to only reflect changes pertaining to this PR/ticket